Cllimate finance explainer (#942)
faktaoklimatu/web-cz: _explainers/klimaticke-finance.md

title:       "Co jsou to klimatické finance?"
slug:        "klimaticke-finance"
published:   2021-10-27
author:      "Alena Moravová"
weight:      75
tags-scopes: [ world ]
tags-topics: [ opatreni, ekonomika ]
cover-source-author: "Harry Strauss"
cover-source-license: "Pixabay License"
cover-source-license-url: "https://pixabay.com/service/license/"
cover-source-text: "Euro Coins Currency Pixabay"
cover-source-url: "https://pixabay.com/photos/euro-coins-currency-cent-euro-cent-1353420/"
perex:       "Pojem \"klimatické finance\" označuje závazek rozvinutých zemí finančně přispívat rozvíjejícím se státům na boj s klimatickou změnou a jejími dopady."

---
Myšlenka klimatických financí vznikla v roce 2009 na klimatické konferenci COP 15 v Kodani. **Zde se 27 rozvinutých států zavázalo, že bude od roku 2020 každoročně posílat 100 miliard amerických dolarů rozvojovým státům**[^1]. Tato myšlenka byla dále stvrzena v Pařížské dohodě (2015) obecnou formulací, že rozvinuté státy mají přispívat rozvojovým zemím v boji s klimatickou změnou. Pařížská dohoda nicméně nemluví o přesné částce ani způsobu určení – rozhodnutí o výši příspěvku je ponecháno na každém jednotlivém státu.

> Tento závazek podpory od rozvinutých k rozvojovým státům je nezbytným krokem pro naplnění klimatických cílů a pro posílení důvěry v globální spolupráci, zároveň však dosud naplňován není, přislíbená částka není dostatečná pro realizaci potřebných opatření a pro transformaci hospodářství v rozvíjejících se zemích, zároveň závazek ani nezohledňuje snižující se kupní sílu dolaru v důsledku inflace.

Reviewer: ```suggestion
Tento závazek podpory od rozvinutých k rozvojovým státům je nezbytným krokem pro naplnění klimatických cílů a pro posílení důvěry v globální spolupráci. Zároveň však dosud naplňován není a ani přislíbená částka není dostatečná pro realizaci potřebných opatření a pro transformaci hospodářství v rozvíjejících se zemích, v neposlední řadě závazek ani nezohledňuje snižující se kupní sílu dolaru v důsledku inflace.
```

## K čemu klimatické finance slouží?

> Rozvojové země mají méně vlastních prostředků a složitější přístup k půjčkám, proto jsou pro ně investice pomáhající zabránit dopadům klimatické změny či se na ně adaptovat složité. Tyto země usilují o hospodářský růst, který by jim pomohl vymýtit chudobu a zlepšit životní úroveň svých obyvatel. **Nejen pro ně, ale i pro nás jako obyvatele rozvinutých a bohatých zemí je klíčové, aby takový růst byl umožněn pomocí čistých technologií.** Pokud by dnešní rozvojové země při svém hospodářském růstu vypouštěly skleníkové plyny v takové míře, jako rozvinuté země vypouštěly při svém růstu v minulosti, pak by dosažení klimatických cílů nebylo možné.

Reviewer: ```suggestion
Rozvojové země mají méně vlastních prostředků a složitější přístup k půjčkám, proto je pro ně složité získat zdroje na investice pro zmenšování či adaptaci na klimatické změny. Tyto země ale zárověň potřebují hospodářský růst, který by jim pomohl vymýtit chudobu a zlepšit životní úroveň jejich obyvatel. **Nejen pro ně, ale i pro nás - obyvatele rozvinutých a bohatých zemí - je klíčové, aby tento růst byl umožněn pomocí čistých technologií.** Pokud by dnešní rozvojové země při svém hospodářském růstu vypouštěly skleníkové plyny v takové míře, jako rozvinuté země vypouštěly při svém růstu v minulosti, pak by dosažení klimatických cílů nebylo možné. Rozvojové země zároveň často patří k těm nejvíce zasaženým klimatickou změnou, část klimatických financí je proto určena i na <glossary id="adaptace">adaptační opatření</glossary>. 
```

> Klimatické finance umožňující rozvojovým zemím investice do čistých technologií jsou velice důležité pro všechny země, nikoli pouze pro jejich příjemce. Rozvojové země často patří k těm nejvíce zasaženým klimatickou změnou, část klimatických financí je proto určena i na <glossary id="adaptace">adaptační opatření</glossary>. Nejde však pouze o finanční přínos, **klimatické finance zároveň podporují snahy těchto zemí s klimatickou změnou něco dělat a posilují důvěru ve spolupráci na globální úrovni.**

Reviewer: ```suggestion
Nejde však pouze o finanční přínos, **klimatické finance zároveň podporují snahy těchto zemí s klimatickou změnou něco dělat a posilují důvěru ve spolupráci na globální úrovni.**
```

Author: to není opakování, má to jiný význam - ale rozumím, že ohledně toho musím být explicitnější

Author: aha, tys to prehodila nahoru, nevsiml jsem si predtim. promyslim to.

## Daří se závazek naplňovat?

> Celková částka sice postupně stoupá, nicméně **v roce 2020 závazek naplněn nebyl** (přesná čísla pro tento rok zatím nejsou k dispozici, ale vzhledem k tomu, že **o rok dříve bylo plnění přibližně 80 % – 79,6 miliard dolarů** – z původně zamýšlené částky, není příliš pravděpodobné, že by bylo částky dosaženo)[^2]. To má velký vliv na důvěru rozvojových států k těm rozvinutým a je to jedno z hlavních témat klimatické konference COP 26 v Glasgow. Závazek bude pravděpodobně prodloužen do roku 2025, přičemž diskuze se povedou ohledně jeho velikosti od roku 2025 dál. Zároveň Velká Británie jako předsedající země vydala nový plán dosažení závazků, který počítá s tím, že by k jeho splnění mělo dojít v roce 2023[^8]. Posuzování naplnění závazku je komplikováno také nejednoznačným vymezením, co klimatické finance přesně zahrnují.

Reviewer: ```suggestion
Celková částka sice postupně stoupá, nicméně **v roce 2020 závazek naplněn nebyl** (přesná čísla pro tento rok zatím nejsou k dispozici, ale vzhledem k tomu, že **o rok dříve bylo plnění přibližně 80 % – 79,6 miliard dolarů** – z původně zamýšlené částky, není příliš pravděpodobné, že by bylo částky dosaženo)[^2]. To má velký vliv na důvěru rozvojových států k těm rozvinutým a je to jedno z hlavních témat klimatické konference COP 26 v Glasgow. Závazek bude pravděpodobně prodloužen do roku 2025, přičemž diskuze se povedou ohledně jeho velikosti od roku 2025 dál. Velká Británie jako předsedající země vydala nový plán dosažení závazků, který počítá s tím, že by k jeho splnění mělo dojít v roce 2023[^8]. Posuzování naplnění závazku je komplikováno také nejednoznačným vymezením, co klimatické finance přesně zahrnují.
```

    source-text="Climate Finance Provided and Mobilised by Developed Countries, OECD"
    source-url="https://www.oecd.org/env/climate-finance-provided-and-mobilised-by-developed-countries-aggregate-trends-updated-with-2019-data-03590fb7-en.htm?"
%}

## Kdo jsou přispěvatelé a kdo příjemci?

**Přispěvateli jsou rozvinuté země světa**, což je v současné době 38 států a patří sem například celá Evropská unie, USA, Kanada, Japonsko, Austrálie a další[^2]. **Příjemci jsou naopak rozvojové země**, kam spadá Indie, Bangladéš, většina států Afriky, ale třeba i Čína. Takzvané transformativní ekonomiky (například Rusko) nespadají ani do jedné kategorie, Pařížská dohoda je ale vyzývá, aby dle svých možností přispívaly také.

<!--
{% include figure.html
    name="prijemci-podle-regionu.png"
    name-mobile="prijemci-podle-regionu.png"
    alt="Příjemci klimatických financí podle regionu."
    source-text="Climate Finance Provided and Mobilised by Developed Countries, OECD"
    source-url="https://www.oecd.org/env/climate-finance-provided-and-mobilised-by-developed-countries-aggregate-trends-updated-with-2019-data-03590fb7-en.htm?"
%}
-->

## Co všechno spadá do klimatických financí?

> Definice klimatických financí je velmi široká – patří sem **finance z bilaterálních i multilaterálních zdrojů, ale například i soukromé zdroje** (ty mohou být započítány do celkového součtu, pokud byly mobilizovány díky veřejným financím). Zároveň se zahrnují nejen granty (tvořily 27 % veřejných klimatických zdrojů v roce 2019), ale také půjčky (71 %) a další zdroje[^2]. Rovněž mobilizované soukromé zdroje mohou mít různé formy, v roce 2019 se jednalo o 14 miliard dolarů ze soukromých zdrojů, tedy o 18 % celkových klimatických financí.

Reviewer: Jsme si teda jisti temi procenty? (21%, 71%) Ja myslela, ze je nakonec chceme vynechat - nebo jsme to overili?

Author: jo, je to převzaté z té OECD zprávy

## Kdo má kolik dávat?

**Přesná výše příspěvku stanovena není**, zatím je to na dobrovolné bázi. Nicméně je tu snaha pozitivně motivovat a jsou vyzdvihovány státy, které plní dostatečně (například Japonsko či Německo)[^6]. Obecně se má za to, že příspěvek by měl být úměrný HDP daného státu a zároveň množství emisí, které tato země produkuje[^7].

{% include figure.html
    name="finance-v-roce-2018.svg"
    name-mobile="finance-v-roce-2018.svg"
    alt="Klimatické finance v roce 2018 ve srovnání s férovým podílem podle HDP."
    source-text="Carbon Brief, World Bank, vlastní výpočty"
    source-url="https://www.carbonbrief.org/analysis-why-climate-finance-flows-are-falling-short-of-100bn-pledge"
%}

Graf ukazuje skutečné příspěvky do klimatických financí v roce 2018 pro vybrané země a srovnává je s férovým podílem podle HDP, který by byl potřeba k dosažení celkového cíle 100 miliard dolarů (za předpokladu stejného podílu mobilizovaných soukromých zdrojů – tedy pro dosažení cíle by postačovalo 70 miliard dolarů z veřejných zdrojů)[^3].

> **Nejvíce do klimatických financí přispívá Japonsko, Německo a Francie**, tyto země zároveň přispívaji přibližně dvojnásobek oproti tomu, kolik by odpovídalo poměru podle výše jejich HDP. Německo přispívá přibližně polovinu částky formou grantů a polovinu formou půjček, oproti tomu u Japonska a Francie půjčky výrazně převažují. USA sice poskytuje většinu financí formou grantu, ale celková částka je velice nízká. Pro dosažení férového podílu podle HDP by USA muselo navýšit své příspěvky na šestinásobek, zároveň USA také patří mezi největší světové znečišťovatele, a to jak v absolutních číslech, tak při přepočtu na obyvatele. **Většina ostatních rozvinutých zemí také přispívá méně, než kolik by odpovídalo jejich férovému podílu.**

Reviewer: ```suggestion
**Nejvíce do klimatických financí přispívá Japonsko, Německo a Francie**, tyto země zároveň přispívaji přibližně dvojnásobek oproti tomu, kolik by odpovídalo poměru podle výše jejich HDP. Německo přispívá přibližně polovinu částky formou grantů a polovinu formou půjček, oproti tomu u Japonska a Francie půjčky výrazně převažují. USA sice poskytuje většinu financí formou grantu, ale celková částka je velice nízká. Pro dosažení férového podílu podle HDP by USA muselo navýšit své příspěvky na šestinásobek, zároveň USA také patří mezi největší světové znečišťovatele, a to jak v absolutních číslech, tak i při přepočtu na obyvatele. **Většina ostatních rozvinutých zemí také přispívá méně, než kolik by odpovídalo jejich férovému podílu.**
```

## Jak je na tom Česká republika?

> Jako rozvinutá země a součást Evropské unie do klimatických financí přispíváme také, a to jak multilaterálními, tak bilaterálními smlouvami. V roce 2018 se jednalo o 185 milionů korun a **ve srovnání s výši našeho HDP jsme na tom ještě výrazně hůře než USA**[^4]. Pro dosažení férového podílu podle HDP bychom v roce 2018 museli přispět 7,8 miliardy korun ročně (přibližně 0,6 % českého státního rozpočtu), navíc také patříme k větším znečišťovatelům při přepočtu emisí na obyvatelu. Příspěvky vycházející z bilaterálních vztahů míří z ČR k některým balkánským státům a dalším zemím (např. Etiopie nebo Kambodža), prostřednictvím multilaterárních fondů jsme v roce 2018 přispěli 26 miliard korun. OECD ve své databázi rozvojové pomoci související s klimatem uvádí vyšší částku 240 milionů korun[^9], ani tato však zdaleka neodpovídá výši závazku rozvinutých zemí.

Reviewer: Tohle je matouci, nevim co se tim snazime rict:
> Příspěvky vycházející z bilaterálních vztahů míří z ČR k některým balkánským státům a dalším zemím (např. Etiopie nebo Kambodža), prostřednictvím multilaterárních fondů jsme v roce 2018 přispěli 26 miliard korun. OECD ve své databázi rozvojové pomoci související s klimatem uvádí vyšší částku 240 milionů korun[^9], ani tato však zdaleka neodpovídá výši závazku rozvinutých zemí.

> Jako rozvinutá země a součást Evropské unie do klimatických financí přispíváme také, a to jak multilaterálními, tak bilaterálními smlouvami. V roce 2018 se jednalo o 185 milionů korun a **ve srovnání s výší našeho HDP jsme na tom ještě výrazně hůře než USA**[^4]. Pro dosažení férového podílu podle HDP bychom v roce 2018 museli přispět 7,8 miliardy korun ročně (přibližně 0,6 % českého státního rozpočtu), navíc také patříme k větším znečišťovatelům při přepočtu emisí na obyvatelu. Příspěvky vycházející z bilaterálních vztahů míří z ČR k některým balkánským státům a dalším zemím (např. Etiopie nebo Kambodža), prostřednictvím multilaterárních fondů jsme v roce 2018 přispěli 26 miliard korun. OECD ve své databázi rozvojové pomoci související s klimatem uvádí vyšší částku 240 milionů korun[^9], ani tato však zdaleka neodpovídá výši závazku rozvinutých zemí.

Reviewer: Ha, mne se tu ztratil koment.. :)
Tehle casti vubec nerozumim:

> Příspěvky vycházející z bilaterálních vztahů míří z ČR k některým balkánským státům a dalším zemím (např. Etiopie nebo Kambodža), prostřednictvím multilaterárních fondů jsme v roce 2018 přispěli 26 miliard korun. OECD ve své databázi rozvojové pomoci související s klimatem uvádí vyšší částku 240 milionů korun[^9], ani tato však zdaleka neodpovídá výši závazku rozvinutých zemí.

Author: ok, opravil jsem a zjednodusil
